recipes/libgd/meta.yaml

```diff
@@ -1,6 +1,6 @@
 package:
   name: libgd
-  version: 2.1.1
+  version: 2.1.1post
```

Comment:
2.1.1bioconda does not work. It is shadowed by libgd-2.1.1.

From travis OS X build:
```
    package                    |            build
    ---------------------------|-----------------
    pkg-config-0.28            |                0         239 KB
    libwebp-0.5.0              |                0         490 KB
    libgd-2.1.1                |                0         297 KB
    ------------------------------------------------------------
                                           Total:         1.0 MB
The following NEW packages will be INSTALLED:
    freetype:   2.5.5-0 
    jbig:       2.1-0   
    jpeg:       8d-2    
    libgd:      2.1.1-0 
    libpng:     1.6.17-0
    libtiff:    4.0.6-1 
    libwebp:    0.5.0-0 
    llvm:       3.3-0   
    pkg-config: 0.28-0  
    xz:         5.0.5-1 
    zlib:       1.2.8-1
```